core/lib/tracehouse/trace-processor.js

```diff
@@ -890,10 +889,10 @@ class TraceProcessor {
     if (!firstMeaningfulPaint) {
       const fmpCand = 'firstMeaningfulPaintCandidate';
       fmpFellBack = true;
-      log.verbose('trace-of-tab', `No firstMeaningfulPaint found, falling back to last ${fmpCand}`);
+      log.verbose(`No firstMeaningfulPaint found, falling back to last ${fmpCand}`);
       const lastCandidate = frameEvents.filter(e => e.name === fmpCand).pop();
       if (!lastCandidate) {
-        log.verbose('trace-of-tab', 'No `firstMeaningfulPaintCandidate` events found in trace');
```

Comment:
Instead of removing this, we could change it to `TraceProcessor` to match other log statements:
https://github.com/GoogleChrome/lighthouse/blob/2f67fbc489fe871054633413622620a794414350/core/gather/gatherers/image-elements.js#L331